Patch to make it easy to sed in the configured python interpeter and mention the liquidctl use flag in the error message.
diff --git a/coolercontrold/src/repositories/liquidctl/liqctld_service.rs b/coolercontrold/src/repositories/liquidctl/liqctld_service.rs
index d5ab5aae..e726560c 100644
--- a/coolercontrold/src/repositories/liquidctl/liqctld_service.rs
+++ b/coolercontrold/src/repositories/liquidctl/liqctld_service.rs
@@ -74,7 +74,7 @@ async fn run_python(script: &[u8], run_token: CancellationToken) -> Result<()> {
         info!("Running liqctld inside an AppImage");
         (format!("{appdir}/usr/bin/python3"), "-I")
     } else {
-        ("python3".to_string(), "-q")
+        ("@@PYTHON@@".to_string(), "-q")
     };
     let mut child = Command::new(cmd)
         .envs(default_env)
diff --git a/coolercontrold/src/repositories/liquidctl/liquidctl_repo.rs b/coolercontrold/src/repositories/liquidctl/liquidctl_repo.rs
index 230d6391..1dca6180 100644
--- a/coolercontrold/src/repositories/liquidctl/liquidctl_repo.rs
+++ b/coolercontrold/src/repositories/liquidctl/liquidctl_repo.rs
@@ -81,10 +81,10 @@ impl LiquidctlRepo {
         }
         if let Err(err) = liqctld_service::verify_env().await {
             let msg = format!(
-                "Python liquidctl system package not detected. If you want liquidctl device \
-                support, please make sure the liquidctl package is installed with your \
-                distribution's package manager. If not, you may disable liquidctl support \
-                to no longer see this message. {err}"
+                "Python liquidctl system package not detected or liquidctl use flag is disabled \
+                for coolercontrold. If you want liquidctl device support, please make sure \
+                the liquidctl use flag is enabled for coolercontrold. If not, you may disable \
+                liquidctl support to no longer see this message. {err}"
             );
             return Err(InitError::PythonEnv { msg }.into());
         }

